--- wesnoth-1.2/configure.ac~	2006-12-22 23:33:48.000000000 +0100
+++ wesnoth-1.2/configure.ac	2006-12-24 13:03:20.000000000 +0100
@@ -106,8 +106,8 @@
 		esac])
 AC_SUBST([DATADIR])
 
-#LOCALEDIR="$datadir/locale"
-LOCALEDIR=translations
+LOCALEDIR="$datadir/locale"
+#LOCALEDIR=translations
 AC_ARG_WITH([localedir],
             AS_HELP_STRING([--with-localedir@<:@=DIR@:>@], [install locale data under dir @<:@translations@:>@]),
 	      [case "${withval}" in
