--- wesnoth-1.8/src/filesystem.cpp~	2010-01-30 16:43:33.000000000 +0000
+++ wesnoth-1.8/src/filesystem.cpp	2010-03-27 14:29:10.983114991 +0000
@@ -288,22 +288,7 @@
 
 std::string get_intl_dir()
 {
-#ifdef _WIN32
-	return get_cwd() + "/translations";
-#else
-
-#ifdef USE_INTERNAL_DATA
-	return get_cwd() + "/" LOCALEDIR;
-#endif
-
-#if HAS_RELATIVE_LOCALEDIR
-	std::string res = game_config::path + "/" LOCALEDIR;
-#else
-	std::string res = LOCALEDIR;
-#endif
-
-	return res;
-#endif
+	return "/usr/share/locale";
 }
 
 std::string get_screenshot_dir()
