--- wesnoth-1.1.10/src/Makefile.am~	2006-10-04 21:42:33.000000000 +0200
+++ wesnoth-1.1.10/src/Makefile.am	2006-10-04 21:43:45.000000000 +0200
@@ -499,7 +499,6 @@
 if SERVER
 install-exec-hook:
 	mkdir -p -m 0700 $(DESTDIR)$(fifodir)
-	-chown $(serveruid):$(servergid) $(DESTDIR)$(fifodir)
 
 uninstall-hook:
 	rm -rf $(DESTDIR)$(fifodir)
--- wesnoth-1.1.10/src/Makefile.in~	2006-10-04 21:42:33.000000000 +0200
+++ wesnoth-1.1.10/src/Makefile.in	2006-10-04 21:44:43.000000000 +0200
@@ -1487,7 +1487,6 @@
 
 @SERVER_TRUE@install-exec-hook:
 @SERVER_TRUE@	mkdir -p -m 0700 $(DESTDIR)$(fifodir)
-@SERVER_TRUE@	-chown $(serveruid):$(servergid) $(DESTDIR)$(fifodir)
 
 @SERVER_TRUE@uninstall-hook:
 @SERVER_TRUE@	rm -rf $(DESTDIR)$(fifodir)
